README.md

```diff
@@ -32,18 +36,25 @@ export LOADTESTING_API_KEY="FIRETEXT_SIMULATION_KEY"
 export FIRETEXT_API_KEY="FIRETEXT_ACTUAL_KEY"
 export STATSD_PREFIX="FAKE_PREFIX"
 export NOTIFICATION_QUEUE_PREFIX="PREFIX-TO-IDENTIFY-SQS-QUEUE"
+export AWS_REGION=<YOUR AWS REGION>
+export AWS_ACCESS_KEY_ID=<YOUR ACCESS KEY ID>
+export AWS_SECRET_ACCESS_KEY=<YOUR SECRET ACCESS KEY>
 "> environment.sh
 ```
 
 NOTE: The SECRET_KEY and DANGEROUS_SALT should match those in the [notifications-admin](https://github.com/alphagov/notifications-admin) app.
 
 NOTE:  Also note the  unique prefix for the queue names. This prevents clashing with others queues in shared amazon environment and using a prefix enables filtering by queue name in the SQS interface.
 
-Install Postgresql
+Install [Postgres.app](http://postgresapp.com/). You will need admin on your machine to do this.
+
+Assuming you downloaded the ZIP file to `~/Downloads` and expanded it there, you should be able to run the following to install the app:
 
-```shell
-    brew install postgres
 ```
+sudo -b ~/Downloads/Postgres.app/Contents/MacOS/Postgres
+```
+
+This will execute the app as a background process (so you can close the terminal window and it won't stop). The install process should offer to move the app to `Applications`.
```

Comment:
Sure. I think you might be right. I was feeling a bit thorough when I wrote that! :)